daemon/main.go

```diff
@@ -178,6 +178,7 @@ func init() {
 	flags.BoolVar(&config.KeepTemplates, "keep-templates", false,
 		"Do not restore template files from binary")
 	flags.StringVar(&config.RunDir, "state-dir", defaults.RuntimePath, "Path to directory to store runtime state")
+	flags.StringVar(&config.LibDir, "lib-dir", defaults.LibraryPath, "Path to directory to store library files")
```

Comment:
library files is misleading, let's do:
"Path to store runtime build environment"